### PR TITLE
Add Chinese strings

### PR DESCRIPTION
**Pull request by felixqu**
Add Chinese strings

### CODE REVIEW

File: src/_locales/zh/messages.json [hunk 1]
```diff
@@ -0,0 +1,450 @@
+{
```
Comment (kspearrin) [inline src/_locales/zh/messages.json]: Is this for traditional Chinese? The file also seems incomplete.

**Comment by felixqu** [comment]
Sorry, I've upload the wrong file "zh/messages.json" by mistaken. The file "zh_CN/messages.json" is corrected.

**Comment by kspearrin** [comment]
@felixqu Are you planning on updating this?

**Comment by kspearrin** [comment]
Going to close this for now since it needs works.

**Pull request closed without merging**